src/urllib3/poolmanager.py

```diff
@@ -220,7 +253,13 @@ def clear(self):
         """
         self.pools.clear()
 
-    def connection_from_host(self, host, port=None, scheme="http", pool_kwargs=None):
+    def connection_from_host(
```

Comment:
Since HTTPConnectionPool and HTTPSConnectionPool don't have any differences in public APIs does it make sense to have a Union here and below?